README.md

```diff
@@ -95,6 +98,16 @@ $ npm uninstall @emotion/core @emotion/styled @emotion/babel-preset-css-prop
 - Remove `@emotion/core` from Typescript configuration (in tsconfig.json file)
 - Remove `.storybook/webpack.config.js` (file created only for emotion css property support in storybook)
 
+### Removing Examples
+
+- Uninstall npm packages
+
+```sh
+$ npm uninstall qrcode.react @types/qrcode.react
+```
+
+- Remove `examples` folder completely
```

Comment:
did you try to remove the folder and run storybook ? (just in case because there is a configuration related to that folder)

Reply:
nope but ill do it now